pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -40,7 +42,7 @@ in
 , enableSharedExecutables ? false
 , enableSharedLibraries ? !stdenv.hostPlatform.isStatic && (ghc.enableShared or false)
 , enableDeadCodeElimination ? (!stdenv.isDarwin)  # TODO: use -dead_strip for darwin
-, enableStaticLibraries ? !(stdenv.hostPlatform.isWindows or stdenv.hostPlatform.isWasm)
```

Comment:
`or` here was a mistake, since `isWindows` attribute is always present - it actually always equal to `stdenv.hostPlatform.isWindows`. ghcjs platform also misses static library support now.